common/components/common/tags/TagSelect.jsx

```diff
@@ -0,0 +1,71 @@
+// @flow
+
+import React from 'react'
+import Select from 'react-select'
+import _ from 'lodash'
+
+
+
+type Props = {|
+  elementId: string,
+  category: string,
+  value?: string,
+  onSelection: (string) => void
+|};
+type State = {|
+  tags: Array<TagDefinition>,
+  selected: TagDefinition
+|};
+
+/**
+ * Dropdown selector for tags
+ */
+class TagSelect extends React.PureComponent<Props, State> {
```

Comment:
nit: this name feels a little funky, how about `TagDropdown` or at least `TagSelector`

Reply:
Done